### PR TITLE
Migrating Footer Component

### PR DESCRIPTION
**Pull request by hanihusam**
The code reference for this component is from [here](https://github.com/reactjs-id/reactjs.id/blob/master/src/components/layout/Footer.tsx). I saw that the Emotion library was also installed. So, I combined the use of Chakra-UI with `styled` Emotion.

@adibfirman

### CODE REVIEW

**Comment by adibfirman** [comment]
Halo mas, boleh coba implement footernya di file `_app.tsx` kah? biar keliatan gimana tampilannya

**Comment by hanihusam** [comment]
> Halo mas, boleh coba implement footernya di file `_app.tsx` kah? biar keliatan gimana tampilannya

Oke Mas

**Review by adibfirman — APPROVED** [review]
Legit LGTM 💯 tinggal dari mas @zainfathoni atau mas @resir014

File: components/layout/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import * as React from 'react'
+
+import { Stack, Grid, Text } from '@chakra-ui/core'
+import styled from '@emotion/styled'
+
+const Root = styled(Grid)`
+  padding: 36px 24px;
+  color: #888;
+  background-color: #111;
```
Comment (resir014) [inline components/layout/Footer.tsx:9]: Please convert these (padding + colors) to chakra-ui style props.

https://chakra-ui.com/style-props

File: components/layout/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  a {
+    color: #fff;
```
Comment (resir014) [inline components/layout/Footer.tsx:11]: Please use chakra-ui style props instead.

https://chakra-ui.com/style-props

File: components/layout/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  }
+`
+
+const Inner = styled(Stack)`
```
Comment (resir014) [inline components/layout/Footer.tsx:15]: Please convert these to chakra-ui style props.

https://chakra-ui.com/style-props

**Review by resir014 — CHANGES_REQUESTED** [review]
Saya tambahkan review ya. Tolong usahakan pakai style props-nya Chakra UI sebelum di-wrap pake `styled()`.

https://chakra-ui.com/style-props

File: pages/_app.tsx [hunk 4]
```diff
@@ -31,6 +32,7 @@ export default function CustomAppPage({ Component, pageProps }: AppProps) {
         <Box as="main" flexGrow={1}>
           <Component {...pageProps} />
         </Box>
+        <Footer />
```
Comment (adibfirman) [inline pages/_app.tsx:35]: ohh iya btw ini boleh minta tolong pindahin ke `components/Page` aja ya mas, thanks mas 👌

Comment (hanihusam) [reply pages/_app.tsx]: Kok enggak bisa Resolve Conflict ya Mas @adibfirman

**Review by resir014 — APPROVED** [review]

**Comment by resir014** [comment]
@hanihusam Sudah saya bantu buat resolve conflict ya, sebelum dimerge.

Sudah saya hapus juga `package-lock.json`, karena kita pake yarn, **bukan npm**.

**Pull request merged**